code/modules/mob/living/carbon/xenomorph/castes/drone/abilities_drone.dm

```diff
@@ -146,7 +146,11 @@
 /datum/action/xeno_action/enhancement/can_use_action()
 	var/mob/living/carbon/xenomorph/X = owner
 	essence_link_action = X.actions_by_path[/datum/action/xeno_action/activable/essence_link]
-	if(!essence_link_action.existing_link || !essence_link_action.existing_link.was_within_range || essence_link_action.existing_link.stacks < 3)
+	if(!essence_link_action.existing_link)
```

Comment:
You changed a perfectly good line into 3 ? What ?

Reply:
I added an additional var to the check, and it was a really damn long line as a result. Felt prudent to split it off rather than have it look like so:
![image](https://user-images.githubusercontent.com/59634950/196256375-3cb060d9-a0cc-406c-b7f1-74c937481b37.png)
_(Note it cuts off due to how long it is.)_